core/src/main/java/org/springframework/security/authentication/ReactiveAuthenticationManagerAdapter.java

```diff
@@ -50,4 +54,13 @@ public ReactiveAuthenticationManagerAdapter(AuthenticationManager authentication
 			})
 			.filter( a -> a.isAuthenticated());
 	}
+
+	/**
+	 * Set a scheduler that will be published on to perform the authentication logic.
+	 * @param scheduler a scheduler to be published on
+	 */
+	public void setScheduler(Scheduler scheduler) {
```

Comment:
Please add a null check and a test to validate the null check

Reply:
Updated the PR with null check and test